packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -790,6 +791,7 @@ function commitLifeCycles(
     case ScopeComponent:
     case OffscreenComponent:
     case LegacyHiddenComponent:
+    case CacheComponent:
```

Comment:
It doesn't have any side-effects anymore so it doesn't need to be in any of these lists. It should fall through to the error.